blueman/plugins/applet/Menu.py

```diff
@@ -4,18 +4,94 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
-from blueman.Functions import *
+import dbus
```

Comment:
Signals work with your fix 🎉, still we should add ``import dbus.service`` as it contains the decorators, we should not rely on other modules for them 😄